plugins/GSdx/GSDevice11.cpp

```diff
@@ -307,7 +307,7 @@ bool GSDevice11::Create(GSWnd* wnd)
 
 	memset(&sd, 0, sizeof(sd));
 
-	sd.Filter = sd.Filter = !!theApp.GetConfig("AnisotropicFiltering", 0) && !theApp.GetConfig("paltex", 0) ? D3D11_FILTER_ANISOTROPIC : D3D11_FILTER_MIN_MAG_MIP_LINEAR;
+	sd.Filter = sd.Filter = theApp.GetConfig("MaxAnisotropy", 1)!=1 && !theApp.GetConfig("paltex", 0) ? D3D11_FILTER_ANISOTROPIC : D3D11_FILTER_MIN_MAG_MIP_LINEAR;
```

Comment:
I think it is more readable to change  m_gs_max_anisotropy range value. It is more logical to have it off at 0 rather than 1.